views/layout.html.twig

```diff
@@ -20,68 +20,98 @@
+                           href="{{ path('docs.view', {page: '00-intro.md'}) }}"
+                           title="Getting started with Composer"
+                           aria-label="Getting started with Composer">Getting Started</a>
+                        <a class="{{ page == 'download' ? 'active':'' }}"
+                           href="{{ path('download') }}"
+                           title="Go the the Download page to see how to download Composer"
+                           aria-label="Go the the Download page to see how to download Composer">Download</a>
+                        <a class="{{ page == 'docs' ? 'active':'' }}"
+                           href="{{ path('docs') }}"
+                           title="View the Composer documentation"
+                           aria-label="View the Composer documentation">Documentation</a>
+                        <a class="last"
+                           href="https://packagist.org/"
+                           title="Browse Composer packages on packagist.org (external link to Packagist.org)"
+                           aria-label="Browse Composer packages on packagist.org (external link to Packagist.org)">Browse
+                            Packages</a>
+                    {% endspaceless %}
                 {% endblock %}
             </header>
-            <div id="main" role="main">
-                {% block content %}
-                {% endblock %}
-            </div>
+            <main role="main">
+                <div id="main">
+                    {% block content %}
+                    {% endblock %}
+                </div>
+            </main>
             <footer>
                 {% block footer %}
                 {% endblock %}
 
-                <p class="license">Composer and all content on this site are released under the <a href="https://github.com/composer/composer/blob/master/LICENSE">MIT license</a>.</p>
+                <p class="license">
+                    Composer and all content on this site are released under the <a
+                            href="https://github.com/composer/composer/blob/master/LICENSE"
+                            title="View the MIT license (external link to GitHub.com)"
+                            aria-label="View the MIT license (external link to GitHub.com)"
+                    >MIT license</a>.
+                </p>
             </footer>
         </div>
 
         <script src="{{ app.request.basePath }}/js/libs/prism.js?v=6"></script>
         <script>
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+            (function (i, s, o, g, r, a, m) {
```

Comment:
this should be reverted. The GA snippet should be kept unedited.

Reply:
Well, more than maybe, but still: the generated syntax by Google will get auto-rewritten by any default refactoring under VSC / PHPStorm. Reverting this.

Don't think that would hurt, though. ;)